larky/src/main/resources/vgs/nts.star

```diff
@@ -25,7 +25,7 @@ CRYPTOGRAM_SUPPORTING_PSP_TYPES = {
 }
 PUSH_ACCOUNT_RECEIPT_PAYLOAD_KEY = "push_account_receipt"
 PUSH_ACCOUNT_DATA_PAYLOAD_KEY = "push_account_data"
-MASTERCARD_PUSH_ACCOUNT_RECEIPT_PREFIX = "MCC-STL-"
+MASTERCARD_PUSH_ACCOUNT_RECEIPT_REGEX = re.compile(r'^(.+)-[A-Z]{3}-(.+)$')

```

Comment:
Can we get a comment explaining the regex/showing some examples it would match?  A lot of folks are intimidated by regex, so having the pattern completely exemplified would help them when they come across this.

You can also point to the test file where there are some examples of it.